contracts/Safe.sol

```diff
@@ -297,7 +297,7 @@ contract Safe is
         // we can safely ignore this malleability.
         bytes32 s;
         uint256 i;
-        for (i = 0; i < requiredSignatures; i++) {
+        for (i = 0; i < requiredSignatures; ++i) {
```

Comment:
Ahhh - this is because of no optimizations in the Safe contracts. I think they _should_ all be the same with optimizations enabled.